### PR TITLE
docker-machine: install completion scripts

### PR DESCRIPTION
**Pull request by leedm777**
docker-machine 0.5.0 introduces some new bash completion scripts. This patch installs them.

### CODE REVIEW

File: Library/Formula/docker-machine.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ class DockerMachine < Formula
 
   depends_on "go" => :build
   depends_on "automake" => :build
+  depends_on "bash-completion"
```
Comment (DomT4) [inline Library/Formula/docker-machine.rb]: Don't need this dep.

**Comment by leedm777** [comment]
@DomT4 Removed the dep. Thanks!

File: Library/Formula/docker-machine.rb [hunk 2]
```diff
@@ -24,6 +24,10 @@ def install
 
     system "make", "build"
     bin.install Dir["bin/*"]
+
+    bash_completion.install "contrib/completion/bash/docker-machine.bash"
+    bash_completion.install "contrib/completion/bash/docker-machine-prompt.bash"
+    bash_completion.install "contrib/completion/bash/docker-machine-wrapper.bash"
```
Comment (bfontaine) [inline Library/Formula/docker-machine.rb]: You could shorten this into one line: `bash_completion.install Dir["contrib/completion/bash/*.bash"]`

Comment (leedm777) [inline Library/Formula/docker-machine.rb]: @bfontaine I don't think that command accepts wildcards.

```
$ brew reinstall --build-from-source ./Library/Formula/docker-machine.rb 
==> Reinstalling docker-machine
==> Downloading https://github.com/docker/machine/archive/v0.5.0.tar.gz
Already downloaded: /Library/Caches/Homebrew/docker-machine-0.5.0.tar.gz
==> make build
Error: No such file or directory - contrib/completion/bash/*.bash
```

Comment (bfontaine) [inline Library/Formula/docker-machine.rb]: Did you use `Dir[]`? It shouldn’t be `bash_install "contrib/completion/bash/*.bash"` but `bash_install Dir["contrib/completion/bash/*.bash"]`.

Comment (leedm777) [inline Library/Formula/docker-machine.rb]: Oh, that's what I did wrong. Fixed and pushed. Thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**